covenant/covenant.go

```diff
@@ -499,6 +610,60 @@ func AcceptDelegationToSign(
 	if !isInCommittee {
 		return false, nil
 	}
+	// 3. For stake expansion, verify if the covenant is in the committee of the previous active delegation
+	if del.IsStakeExpansion() {
```

Comment:
One thing that I was thinking about is, what if the covenant that is running the covenant emulator is not in the current covenant committee set, but only in the committee of the previous staking parameters 

In theory, he could still sign for the spend of the previous staking, but not being in the list for creation of the new BTC stake expansion tx

Reply:
cc @KonradStaniec

Reply:
humm, so we won't cover the case if the covenant that was rotated in the currently covenant set, but was still running their covenant signer

Reply:
Yep we won't cover this. Imo there are two reasons it is not worth it:
- most possibly any rational operator after being excluded from the committe will stop their infra.
- to cover this use case we would need to have a way to separate stake expansion signature message from the more general `MsgAddCovenantSigs` , as the roteted covenant member won't be able to sign for new unbonding/slashing/slashing_unbonding transactions. This brings a lot of complexity (additional message/special handling) for pretty niche use case.